plothist/plotters.py

```diff
@@ -495,7 +495,14 @@ def plot_hist_difference(hist1, hist2, ax, **kwargs):
 
 
 def compare_two_hist(
```

Comment:
Same comments. Shall we try to optimise and have a single function? In that case, hist_1 and hist_2 are not symmetric. One is data and the other one is prediction (or reference). Then we could use call this function in `hep_plotters`